ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -868,6 +857,27 @@ class JITModBuilder {
     }
   }
 
+  GlobalVariable* cloneGlobalVariable(Value *V) {
+    // If there's a reference to a GlobalVariable, copy it over to the
```

Comment:
I'd kill this comment. There's no "if" about it now. `V` is assumed to be a global. Add an assertion?

Reply:
Ah, this was meant to go to the part that calls this. I will move it.

Reply:
Actually, you are right. This can just be deleted: [764720c](https://github.com/ykjit/yk/pull/563/commits/764720c21768eb4217f1991db15e2d6fa991066f)